rankings.py

```diff
@@ -89,7 +92,7 @@ def generate_ranking(dao, now=datetime.now(), day_limit=60, num_tourneys=2, tour
         id=ObjectId(),
         region=dao.region_id,
         time=now,
-        tournaments=[t.id for t in tournaments],
+        tournaments=[t.id for t in ranking_tournaments],
```

Comment:
Previous code left in ALL tournaments that have ever happened. This was also affected in the database.